app/src/main/java/com/codesses/priceconverter/common/utils/AppConstants.kt

```diff
@@ -0,0 +1,6 @@
+package com.codesses.priceconverter.common.utils
+
+object AppConstants {
+    const val BIT_PAY_BASE_URL = "https://api.coingecko.com/"
```

Comment:
Those constants should be part of the "plugin" code. It should be easy to add new price sources

Reply:
See current implementation, every source of rates/prices has own class that shares interface